packages/models-library/src/models_library/api_schemas_webserver/socketio.py

```diff
@@ -0,0 +1,17 @@
+from ..users import GroupID, UserID
+
+
+class SocketIORoom(str):
+    __slots__ = ()
+
+    @classmethod
+    def from_socket_id(cls, socket_id: str) -> "SocketIORoom":
```

Comment:
I think this should be `from_str`, right? Otherwise the annotation should be `def from_socket_id(cls, socket_id: "SocketIORoom") -> "SocketIORoom"` which is some sort of casting